bookmarks/account/forms.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.models import User
+from django import forms
+
+
+class UserRegistrationForm(forms.ModelForm):
+    password = forms.CharField(label="Password", widget=forms.PasswordInput)
+    password2 = forms.CharField(
+        label="Repeat password", widget=forms.PasswordInput
+    )
+
+    class Meta:
+        model = User
+        fields = ("username", "first_name", "email")
+
+    def clean_password2(self):
+        cd = self.cleaned_data

```

Comment:
Use self.cleaned_data directly. What that cd even mean?
![image](https://user-images.githubusercontent.com/9105503/207555891-45953050-db69-48e8-8307-a058746aa964.png)